Formula/swiftplate.rb

```diff
@@ -0,0 +1,59 @@
+class Swiftplate < Formula
+  desc "Cross-platform Swift framework templates from the command-line"
+  homepage "https://github.com/JohnSundell/SwiftPlate"
+  url "https://github.com/JohnSundell/SwiftPlate/archive/1.2.1.tar.gz",
```

Comment:
You don't need the `:tag` here - Homebrew will figure out the version from the URL.